hosts

```diff
@@ -1,40 +1,5 @@
-[collabora]
-master
-builder01
-builder02
-terbium
-
-[linaro]
-builder03
-builder04
-builder05
-builder06
-builder07
-builder08
-builder09
-builder10
-garagebuilders01
-garagebuilders02
-garagebuilders03
-garagebuilders04
-garagebuilders05
-
-[baylibre]
-builder-baylibre01
-
-[azure]
-azure-kernelci1
-azure-kernelci2
-azure-kernelci3
-azure-kernelci4
-azure-kernelci6
-azure-kernelci7
-azure-kernelci9
-azure-kernelci-jenkins-usa
-azure-kernelci-jenkins-europe
+[kernelci-prod-jenkins-runner-4]
+kernelci-prod-jenkins-runner-4.eastus.cloudapp.azure.com ansible_user=kernelci
 
 [kernel-builders:children]
-collabora
-linaro
-baylibre
-azure
+kernelci-prod-jenkins-runner-4
```

Comment:
Ideally we should be adding runners 1, 2 and 3 as well.  At least 1 and 3 are online, 2 is currently offline but can be powered up to run the recipe.  It could be done as a follow-up if it's too complicated to do it as part of this PR.